lib/progress_bar/determinate.ex

```diff
@@ -34,19 +39,25 @@ defmodule ProgressBar.Determinate do
 
   # Private
 
-  defp formatted_percent(false, _), do: ""
-  defp formatted_percent(true, number) do
-    " " <> String.rjust(Integer.to_string(number), 3) <> "%"
-  end
+  defp get_suffix(%{suffix: false, bytes: true}), do: :bytes
```

Comment:
I've implemented it with backward compatibilty, but my original idea was to deprecate the `bytes` option on the next version. WDYT?

Reply:
Good thinking. I'd also be fine with breaking compatibility and bumping the major version. Could then also doc the old syntax next to the new docs, so it's easier to find. And maybe introduce a CHANGELOG. What do you think about that option? I'd be happy to do those things myself as part of making the release.

Reply:
Well, I prefer the new way and less code is always better, so let's do it! Will make the changes here and commit them!

Reply:
As for the CHANGELOG and the docs, if you could help me with them it would be awesome! Since I don't know exactly what's in your mind.

I'll remove the `get_suffix` method what then makes this change backward incompatible. OK?